cmd/suite_test.go

```diff
@@ -1,13 +1,27 @@
 package cmd
 
 import (
+	"os"
+	"path/filepath"
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/artifacts"
 )
 
 func TestCMD(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "CMD Suite")
 }
+
+var artifactsAndLogTemp = func() {
+	tmpDir, err := os.MkdirTemp("", "cmd-execute-*")
+	Expect(err).ToNot(HaveOccurred())
+	artifacts.SetDir(filepath.Join(tmpDir, "artifacts"))
+	DeferCleanup(os.RemoveAll, tmpDir)
+	os.Setenv("PFLT_ARTIFACTS", artifacts.Path())
+	os.Setenv("PFLT_LOGFILE", filepath.Join(tmpDir, "preflight.log"))
+	DeferCleanup(os.Unsetenv, "PFLT_ARTIFACTS")
+	DeferCleanup(os.Unsetenv, "PFLT_LOGFILE")
```

Comment:
Same as previous instance of similar code, reorganize defers near each other and do we need to use envs?